wiki/Beatmap/Hype/en.md

```diff
@@ -10,10 +10,10 @@ Every [pending](/wiki/Beatmap/Category#work-in-progress-and-pending) [beatmap](/
 
 ## Ranking requirement
 
-Beatmaps need at least 5 hype (one complete bar of the hype train) to be eligible for [nomination](/wiki/Beatmap_ranking_procedure#nominations), a step towards getting the map ranked.
+Beatmaps need at least 5 hypes (one complete bar of the hype train) to be eligible for [nomination](/wiki/Beatmap_ranking_procedure#nominations), a step towards getting the map ranked.
```

Comment:
```suggestion
Beatmaps need at least 5 hype (one complete bar of the hype train) to be eligible for [nomination](/wiki/Beatmap_ranking_procedure#nominations), a step towards getting the map ranked.
```

Reply:
the previous was incorrect

Reply:
I dunno, calling them "hypes" feels really corny.

When you build hype you don't call it hypes.

🤷

Reply:
Hm.. I think the article should stick to one or the other.. using `5 hype` here and then `10 hypes` in the following paragraph is a bit weird, I think.

Reply:
"hype" is treated as a countable noun here, i.e. instances of hype posts in a beatmap discussion. if you don't want to do that and insist that "hype" can't be countable, you'd write "5 hype posts" or something similar instead